Flight.cpp

```diff
@@ -83,6 +91,7 @@ class DeltaCalculator:public AirlineCalculator{
     DeltaCalculator() = default;
```

Comment:
singleton class should have copy and assign constructor = delete

Reply:
Why is that?
https://www.tutorialspoint.com/Explain-Cplusplus-Singleton-design-pattern